src/main/java/core/basesyntax/service/impl/CsvReaderImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CsvReader;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+public class CsvReaderImpl implements CsvReader {
+    private static final int SKIP_LINES = 1;
+
+    @Override
+    public List<String> readLines(String fileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileName))) {
```

Comment:
Avoid using FileReader directly as it does not support character encoding. Consider using InputStreamReader with a specified charset instead.